README.md

```diff
@@ -1,81 +1,70 @@
-[![License: Apache 2.0](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/dapr/js-sdk/blob/master/LICENSE)
-[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2Fdapr%2Fcomponents-contrib.svg?type=shield)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2Fdapr%2Fcomponents-contrib?ref=badge_shield)
+<p align="center">
+  <a href="https://dapr.io">
+    <img src="https://dapr.io/images/dapr.svg" height="128">
+    <h1 align="center">Dapr</h1>
+  </a>
+</p>
+
+<p align="center">
+  <a aria-label="NPM version" href="https://www.npmjs.com/package/@dapr/dapr">
+    <img alt="" src="https://img.shields.io/npm/v/@dapr/dapr?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="License" href="https://github.com/dapr/js-sdk/blob/master/LICENSE">
+    <img alt="" src="https://img.shields.io/badge/License-Apache_2.0-blue.svg?style=for-the-badge&labelColor=000000">
+  </a>
+  <a aria-label="Join the community on Discord" href="https://discord.com/invite/ptHhX6jc34">
+    <img alt="" src="https://img.shields.io/badge/Join%20the%20community-blueviolet.svg?style=for-the-badge&logo=Discord&labelColor=000000&logoWidth=20&logoColor=FFFFFF">
+  </a>
+</p>
+
+## Getting Started
+
+Instantly get started by installing the Dapr JS SDK and reading the [getting started documentation](https://docs.dapr.io/developing-applications/sdks/js) or [follow one of the quickstarts](https://github.com/dapr/quickstarts)
 
-# Dapr Node.js SDKs
-
-The official [Dapr](https://dapr.io) Node.js SDK that allows interfacing with the Dapr sidecar for easy application building.
-
-## Introduction
-
-The Dapr JS SDK will allow you to interface with the Dapr process that abstracts several commonly used functionalities such as Service-to-Service invocation, State Management, PubSub, and more.
-
-![](./documentation/assets/dapr-architecture.png)
-
-Looking at the illustration above, we can see there will always always be another process (the Dapr Sidecar) running that your application will interface with. This process can either be started manually (e.g. through Dapr CLI) or injected as a container (e.g. through Kubernetes Sidecar injection in your pod).
-
-For your application to interface with this, 2 components should be taken into account:
-* **DaprServer:** Dapr Sidecar -> our Application - When we Subscribe to a Topic, Create an Actor, ... we receive an event from the Dapr process that abstracted its implementation. Our application should thus listen to this (which this SDK helps you with).
-* **DaprClient:** Your Application -> Dapr Sidecar - When we want to Publish an Event, Execute a Binding, ... we will talk with the Dapr process that calls its implementation for us so that we don't have to write it ourself!
-
-## Simple Example
+```
+npm install --save @dapr/dapr
+```
 
-> [Full version](./examples/http/pubsub)
+> ⚠️ the old `dapr-client` package has been deprecated
```

Comment:
nit: 
```suggestion
> ⚠️ the [`dapr-client`](https://www.npmjs.com/package/dapr-client) package has been deprecated. Please see https://github.com/dapr/js-sdk/issues/259 for more.
```

At this point, I also wonder if it will be effective to have a doc with steps on how to migrate to the new NPM package and use that here, as well as in the warning log, social media posts, etc.